### DISCUSSION TITLE
Not able to create new collections and globals, or change existing fields in production

### DISCUSSION THREAD

**Discussion by stefvw93** [payloadcms/payload#10442] [Q&A]
### Describe the Bug

Payload 2.26, mongodb

I am no longer able to add collections and globals, or change existing collections and globals. I can't add or change fields, I can't create entire new collections. The app runs without errors, but the database is not updated when I restart payload or when I save changes in the admin UI.

Local development mode works fine, but it happens only in remote production build.

Has anyone had this problem? Desperate for help here...

### To Reproduce

Create a new collection, add it to payload config; collection is not actually created in the database and does not show up in admin UI.
Create a new global, add it to payload config; global is not added in the database on save, and does not show up in admin UI.

### Payload Version

2.26

### Adapters and Plugins

@payloadcms/db-mongodb@1.4.3

**Comment by DanRibbens** [comment]
> Local development mode works fine, but it happens only in remote production build.

Are you use you've deployed the code with the changes? I assume that the code you're trying to deploy is not making it to your production environment. If you share more information about your CI/CD or deployoment process we can help diagnose and troubleshoot.
